sebs/benchmark.py

```diff
@@ -494,6 +494,10 @@ def build(
             shutil.rmtree(self._output_dir)
         os.makedirs(self._output_dir)
 
+        benchmark = self.benchmark
+        if self._deployment_name == "azure":
```

Comment:
I think we should have one place where we rename the function according to platform and trigger type.